cmd/proxygenerator/interceptor.go

```diff
@@ -147,6 +127,25 @@ func NewPayloadVisitorInterceptor(options PayloadVisitorInterceptorOptions) (grp
 	}, nil
 }
 
+func parseGrpcPayload(ctx context.Context, err error, s *status.Status, inbound *VisitPayloadsOptions) error {
```

Comment:
Pedantic, but this name is a little generic for something only for errors. Maybe `visitGrpcError` or something?